src/api/factcheck.py

```
@@ -1,613 +1,41 @@
+import time
+from models.factcheck import (
+    FactCheckRequest,
+    FactCheckResponse,
+    Claim,
+    Source
+)
+
+router = APIRouter(prefix="/api/v1")
+
+@router.post("/factcheck", response_model=FactCheckResponse)
+async def factcheck(request: FactCheckRequest):
```

Ruff (0.11.9)

Missing docstring in public function

(D103)